niftilib/nifti_tester001.c

```diff
@@ -7,7 +7,7 @@ enum NIFTITEST_BOOL {
   NIFTITEST_FALSE=0
 };
 
-void _PrintTest(const int line,const char * message,const int FailureOccured, const enum NIFTITEST_BOOL isFatal,int *ErrorAccum)
+static void _PrintTest(const int line,const char * message,const int FailureOccured, const enum NIFTITEST_BOOL isFatal,int *ErrorAccum)
```

Comment:
This is a main program, so static should not matter.

Reply:
Indeed, there should be no change in behaviour here.

The motivation is to fix all `-Wmissing-prototypes` warnings because that warning in general can and does find real bugs.

It's just good practice for symbols not needed outside the translation unit to be declared `static`.

Reply:
@seanm Thanks.  I agree with you.  

While I agree that it does not matter to the functioning of the program, it does make maintaining the package more difficult for developers who utilize advanced IDEs and code checkers. Adding static in places where it could be optional makes long-term maintenance easier for new (and some old) developers.

I feel similarly about "const". Using const makes refactoring and other long-term code maintenance concerns less dangerous.  It conveys to new developers a set of constraints that can make understanding the code easier.